@theme/plugins/code-samples.js

```diff
@@ -43,6 +43,7 @@ export function codeSamples() {
 
         actions.createSharedData('code-samples', { codeSamples: sortedSamples, langs: Array.from(allLands) });
         actions.addRouteSharedData('/resources/code-samples/', 'code-samples', 'code-samples');
+        actions.addRouteSharedData('/ja/resources/code-samples/', 'code-samples', 'code-samples');
```

Comment:
Note for the future: if we add more languages to the site, we need to add the shared data to more routes here and in the blog index page, unless/until Redocly updates their plugin API.